tox.ini

```diff
@@ -47,22 +53,12 @@ deps =
     black~={[base]black_version}
     sphinx
 commands =
-    pycodestyle sphinx_bluebrain_theme
-    pydocstyle sphinx_bluebrain_theme
-    pylint sphinx_bluebrain_theme
-    pycodestyle mkdocs2sphinx
-    pydocstyle mkdocs2sphinx
-    pylint mkdocs2sphinx
-    pycodestyle translate_templates.py
-    pydocstyle translate_templates.py
-    pylint translate_templates.py
-    pycodestyle setup.py
-    pydocstyle setup.py
-    pylint setup.py
-    pycodestyle tests
-    pydocstyle tests
-    pylint tests
+    pycodestyle {[base]paths_to_lint}
+    pydocstyle {[base]paths_to_lint}
+    pylint {[base]paths_to_lint}
+
     pylint doc/source/conf.py
```

Comment:
`doc/source/conf.py` could be moved to the line above, so that pylint is executed only once